app-doc/linux-device-drivers/linux-device-drivers-3-r1.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Howto write linux device drivers (updated for Linux 2.6)"
+HOMEPAGE="
+	http://www.oreilly.com/catalog/linuxdrive3/
+	http://lwn.net/Kernel/LDD3/
+"
+SRC_URI="
+	http://lwn.net/images/pdf/LDD3/ldd3_pdf.tar.bz2
+	mirror://gentoo/LDD3-examples.tar.gz
+"
+S="${WORKDIR}"
+
+# original URL is this:
+# http://examples.oreilly.com/linuxdrive3/examples.tar.gz
```

Comment:
I feel like the author hadn't heard of -> ...

If that link is still live, replace the mirror:// junk with this with a rename.